ResetPassword.tsx

```diff
@@ -0,0 +1,170 @@
+    </View>
+
+    <View>
+      <Text
+        style={{
+          marginLeft: 35,
+          paddingBottom: 1,
+          marginBottom: 10,
+          //marginTop: 100,
+        }}
+      >
+        Please enter your new password:
+      </Text>
+    </View>
+    <Formik
+      initialValues={{
+        password: "",
+        passwordConfirmation: "",
+      }}
+      onSubmit={(values, actions) => {
+        alert(JSON.stringify(values));
+
+        actions.resetForm();
+        setTimeout(() => {
+          actions.setSubmitting(false);
+        }, 1000);
+      }}
+      validationSchema={validationSchema}
+    >
+      {(formikProps) => (
+        <React.Fragment>
+          <View style={{ marginHorizontal: 20, marginVertical: 5 }}>
+            <TextInput
+              underlineColorAndroid="transparent"
+              placeholderTextColor="#9a73ef"
+              autoCapitalize="none"
+              placeholder="password"
+              style={styles.input}
+              onChangeText={formikProps.handleChange("password")}
+              onBlur={formikProps.handleBlur("password")}
+              secureTextEntry
+            />
+            <Text style={{ color: "red", paddingLeft: 15 }}>
+              {formikProps.touched.password && formikProps.errors.password}
+            </Text>
+          </View>
+
+          <View style={{ marginHorizontal: 20 }}>
+            <TextInput
+              underlineColorAndroid="transparent"
+              placeholderTextColor="#9a73ef"
+              autoCapitalize="none"
+              placeholder="Confirm password"
+              style={styles.input}
+              onChangeText={formikProps.handleChange("passwordConfirmation")}
+              onBlur={formikProps.handleBlur("passwordConfirmation")}
+              secureTextEntry
+            />
+            <Text
+              style={{
+                color: "red",
+
+                paddingLeft: 15,
+              }}
+            >
+              {formikProps.touched.passwordConfirmation &&
+                formikProps.errors.passwordConfirmation}
+            </Text>
+          </View>
+
+          {formikProps.isSubmitting ? (
+            <ActivityIndicator />
+          ) : (
+            <View style={styles.submitButton}>
+              <Button title="Send" onPress={formikProps.handleSubmit} />
+            </View>
+          )}
+        </React.Fragment>
+      )}
+    </Formik>
+  </SafeAreaView>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    // paddingTop: 100,
+    //  marginBottom: 20,
+    paddingBottom: 60,
+    flex: 1,
+    justifyContent: "center",
+    direction: "inherit",
+
+    //marginLeft: 10,
+  },
+  input: {
+    margin: 15,
+    height: 40,
+    borderColor: "blue",
+    borderWidth: 1,
+    //borderColor:'black',
+  },
+  submitButton: {
+    backgroundColor: "#0000FF",
+    padding: 10,
+    margin: 15,
+    height: 40,
+  },
+  submitButtonText: {
+    color: "white",
+  },
+  appTitleLarge: {
```

Comment:
It seems like there are styles you aren't using here. Please, clean it up if that's the case.